includes/widgets/image-box.php

```diff
@@ -270,12 +270,22 @@ protected function register_controls() {
 			]
 		);
 
+		$this->add_group_control(
+			Group_Control_Border::get_type(),
+			[
+				'name' => 'image_border',
+				'selector' => '{{WRAPPER}} .elementor-image-box-wrapper img',
```

Comment:
Why `.elementor-image-box-wrapper` and not `.elementor-image-box-img`?

Reply:
I used it since you used it in the image_border_radius control.
But I agree it should be .elementor-image-box-img, the changes have been made.